packages/hacl-star-raw/hacl-star-raw.0.4.5+dune/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+synopsis: "Auto-generated low-level OCaml bindings for EverCrypt/HACL*"
+description: """
+This package contains a snapshot of the EverCrypt crypto provider and
+the HACL* library, along with automatically generated Ctypes bindings.
+For a higher-level idiomatic API see the `hacl-star` package, of
+which `hacl-star-raw` is a dependency.
+"""
+maintainer: "Victor Dumitrescu <victor.dumitrescu@nomadic-labs.com>"
+authors: [ "Project Everest" ]
+license: "Apache-2.0"
+homepage: "http://github.com/dune-universe/hacl-star-dune"
+bug-reports: "https://github.com/project-everest/hacl-star/issues"
+depends: [
+  "dune" { >= "2.6" }
+  "ocaml" { >= "4.08.0" }
+  "ocamlfind" {build}
+  "ctypes" { >= "0.18.0" }
+  "conf-which" {build}
+]
+available: [
+  arch != "ppc64" & arch != "ppc32" &
+  arch != "arm32" & arch != "x86_32" &
+  arch != "arm64" & arch != "s390x" &
+  (os = "freebsd" | os-family != "bsd")
+]
+x-ci-accept-failures: [
+  "centos-7" # Default C compiler is too old
+  "oraclelinux-7" # Default C compiler is too old
+]
+build: [
+  ["sh" "-exc" "cd raw && ./configure"]
```

Comment:
Ah, looks like `hacl-star-raw` doesn't use dune to build. I'm assuming we need to create the dune file for this as well since `hacl-star` depends on `hacl-star-raw`?

cc @NathanReb @emillon

Reply:
hacl-star-raw does use dune to build, but both ocaml-ci and opam-monorepo ignore the instructions in the `build:` section. so you can switch these instructions to the dune ones but it's not necessary.
Note that `hacl-star` already uses dune upstream so I don't think it's necessary to submit a port for it.

Reply:
It is necessary or at the very least safer to submit a port for both. Especially since `dune-universe/hacl-star-dune` defines both packages anyway.

Reply:
Using dune in the build instructions would be best here as it would ease the use of opam-ci on this repo so I'd advise to change the build instructions as well.